packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -2116,6 +2117,108 @@ QUnit.module('Events', moduleConfig, () => {
+        { propertyName: 'onResize', propertyValue: () => {} },
+        { propertyName: 'onItemCollapsed', propertyValue: () => {} },
+        { propertyName: 'onItemExpanded', propertyValue: () => {} },
+        { propertyName: 'allowKeyboardNavigation', propertyValue: false },
+        { propertyName: 'allowKeyboardNavigation', propertyValue: true },
+        { propertyName: 'rtlEnabled', propertyValue: false },
+        { propertyName: 'rtlEnabled', propertyValue: true },
+        { propertyName: 'separatorSize', propertyValue: 12 },
+    ].forEach(({ propertyName, propertyValue }) => {
+        QUnit.test(`${propertyName} property should be passed to nested splitter on initialization`, function(assert) {
+            this.reinit({
+                [propertyName]: propertyValue,
+                items: [{
+                    splitter: {
+                        dataSource: [{ text: 'pane 1' }, { text: 'pane 2' }]
+                    }
+                }]
+            });
+
+            const $nestedSplitter = getNestedSplitter(this.$element);
+            const nestedSplitterInstance = $nestedSplitter.dxSplitter('instance');
+
+            assert.strictEqual(nestedSplitterInstance.option(propertyName), propertyValue);
+        });
+    });
+
+    [
+        { propertyName: 'onResizeStart', propertyValue: () => {} },
+        { propertyName: 'onResizeEnd', propertyValue: () => {} },
+        { propertyName: 'onResize', propertyValue: () => {} },
+        { propertyName: 'onItemCollapsed', propertyValue: () => {} },
+        { propertyName: 'onItemExpanded', propertyValue: () => {} },
+        { propertyName: 'allowKeyboardNavigation', propertyValue: false },
+        { propertyName: 'allowKeyboardNavigation', propertyValue: true },
+        { propertyName: 'rtlEnabled', propertyValue: false },
+        { propertyName: 'rtlEnabled', propertyValue: true },
+        { propertyName: 'separatorSize', propertyValue: 12 },
+    ].forEach(({ propertyName, propertyValue }) => {
+        QUnit.test(`${propertyName} property should be passed to nested splitters at runtime`, function(assert) {
+            this.reinit({
+                items: [{
+                    splitter: {
+                        dataSource: [{ text: 'pane 1' }, { text: 'pane 2' }]
+                    }
+                }, {},
+                {
+                    splitter: {
+                        dataSource: [
+                            { text: 'pane 1' },
+                            {
+                                splitter: {
+                                    dataSource: [
+                                        { text: 'pane 1' },
+                                        { text: 'pane 2' }
+                                    ]
+                                }
+                            }
+                        ]
+                    }
+                }]
+            });
+
+            this.instance.option(propertyName, propertyValue);
```

Comment:
I think it's important that the previous options value be declared during the initialization of the component. After changing an option at runtime, events should use the new handler.